eb_hooks.py

```diff
@@ -167,7 +167,7 @@ def parse_hook_openblas_relax_lapack_tests_num_errors(ec, eprefix):
         cfg_option = 'max_failing_lapack_tests_num_errors'
         if get_cpu_architecture() == AARCH64:
             orig_value = ec[cfg_option]
-            ec[cfg_option] = 300
+            ec[cfg_option] = 400
```

Comment:
@casparvl Let's add a comment (an extra comment line above) that refers to https://github.com/EESSI/software-layer/issues/314 here, to make the lives of our future selves a bit easier? ;)

Reply:
Done!